### PR TITLE
STR-101 -- FE: Make dashboard page mobile responsive

### PR DESCRIPTION
**Pull request by BrianLinggadjaja**
# Description
Made desktop version responsive; added specific alt-text depending on substance.
  
# Testing
1. Login as `cyndee.bellamy` or `april.feldman`.
2. View the mobile and desktop viewports and ensure everything scales properly.
3. Check to see if there is any overflows, positioning of elements.
4. Ensure it meets the format of the desktop mock-up.

### CODE REVIEW

**Review by andrewMETALAB — CHANGES_REQUESTED** [review]
Looking really good. The only funkyness I see is:

(1) Some funkyness with the thumbnail images, but I'm not going to worry about it for now, since these are not the final images. Once we have the final images, let's make sure that we resize them in photoshop so that they aren't unnecessarily large. 
(2) Looks like the navbar extends beyond the rest of the content on both sides. Can we fix this please?

**Review by andrewMETALAB — APPROVED** [review]

File: resources/js/components/dashboard/DashboardModule.vue [hunk 1]
```diff
@@ -2,35 +2,46 @@
...
+                    </p>
+                    <loading-progress class="dashboard-module__loader"
+                      :progress="item.completion.progress/100"
+                      :size="size"
+                      :shape="shape.circle"
+                    />
+                  </div>
+                </div>
+                <div class="col-5 col-md-10 text-left">
+                  <span class="dashboard-module__date ml-3">{{ item.completion.date}}</span>
```
Comment (frlzjosh) [inline resources/js/components/dashboard/DashboardModule.vue]: based on the mockup it seems like you can add a `mt-1` class in this div to add some margin to the text that goes under here

**Review by frlzjosh — DISMISSED** [review]
Very minor things:
1 ) The buttons are a bit smaller on the mock up
2 ) The photos have a `border-radius` on the mock up for desktop version
3 ) The titles for the modules are bold on desktop
4 ) For some reason the font awesome icons went away

**Comment by BrianLinggadjaja** [comment]
> Very minor things:
> 1 ) The buttons are a bit smaller on the mock up
> 2 ) The photos have a `border-radius` on the mock up for desktop version
> 3 ) The titles for the modules are bold on desktop
> 4 ) For some reason the font awesome icons went away

1. I followed the font-size from the mock-up which was 20px and scaled it into rem.
2. The updated desktop mock-up doesn't have rounded edges I believe.
3. Font-weight bold wasn't supported in our current font so I just made it Arial in the header.
4. Fixed the font-awesome icon not showing on desktop.

File: resources/js/components/dashboard/DashboardModule.vue [hunk 2]
```diff
@@ -56,10 +70,11 @@ export default {
         alcohol: {
           name: 'Alcohol',
           image: 'images/thumbnail/alcohol_thumbnail.jpg',
+          altText: 'alcohol substance',
```
Comment (frlzjosh) [inline resources/js/components/dashboard/DashboardModule.vue:73]: deng that's hard

**Review by frlzjosh — APPROVED** [review]
you riiight!

**Pull request merged**